scripts/vmbuild.rb

```diff
@@ -37,20 +37,29 @@
 IMGFAC_DIR          = BUILD_BASE.join("imagefactory")
 IMGFAC_CONF         = CFG_DIR.join("imagefactory.conf")
 STORAGE_DIR         = BUILD_BASE.join("storage")
-ISOS_DIR            = BUILD_BASE.join("isos")
+ISOS_DIR            = Pathname.new("/build/isos")
```

Comment:
This should use the BUILD_BASE since it's GHA-aware, right?

```suggestion
ISOS_DIR            = BUILD_BASE.join("isos")
```

Reply:
No, this is for the CentOS install ISO.  We can keep one copy in `/build/isos` rather than a copy in each `BUILD_BASE`.

Reply:
In taht case, let's add a comment here, because it's confusing in the code and looks like a bug.  Perhaps something like

```suggestion
ISOS_DIR            = Pathname.new("/build/isos") # NOTE: We are not using run-specific BUILD_BASE in order to share the iso across builds
```

Reply:
Done